src/utils/navigation.ts

```diff
@@ -39,6 +39,16 @@ export const NAVIGATION: Navigation = {
     previousPage: () => config.SECURE_UPDATE_FILTER_URL,
     nextPage: [config.CONFIRM_OVERSEAS_ENTITY_DETAILS_PAGE]
   },
+  [config.UPDATE_OVERSEAS_ENTITY_CONFIRM_URL]: {
```

Comment:
This was missing so added instead of edited?

Reply:
Yes it seemed to be! I was looking for something to put my new page as the nextPage of but there was no entry for UPDATE_OVERSEAS_ENTITY_CONFIRM_URL, not sure if it was intentional or not, is it ok to add?

Reply:
Yes, locks ok to me.